modules/core/src/smithy4s/server/UnaryServerEndpoint.scala

```diff
@@ -29,6 +29,16 @@ object UnaryServerEndpoint {
       endpoint: Endpoint[Op, I, E, O, SI, SO],
       codecs: UnaryServerCodecs[F, Request, Response, I, E, O],
       middleware: (Request => F[Response]) => (Request => F[Response])
+  )(implicit F: MonadThrowLike[F]): Request => F[Response] = {
+    apply(interpreter, endpoint, codecs, middleware, encodeErrorsBeforeMiddleware = false)
+  }
+
+  def apply[F[_], Op[_, _, _, _, _], Request, Response, I, E, O, SI, SO](
+      interpreter: FunctorInterpreter[Op, F],
+      endpoint: Endpoint[Op, I, E, O, SI, SO],
+      codecs: UnaryServerCodecs[F, Request, Response, I, E, O],
+      middleware: (Request => F[Response]) => (Request => F[Response]),
+      encodeErrorsBeforeMiddleware: Boolean = false
```

Comment:
let's not add a default value : the overload already plays this role and the default value will make it harder to add more parameters down the line, should the need arise